tests/Unit/MissingDataFileTest.php

```diff
@@ -1,20 +1,22 @@
 <?php
 
+namespace DivineOmega\Countries\Tests;
+
 use DivineOmega\Countries\Countries;
 use PHPUnit\Framework\TestCase;
 
 final class MissingDataFileTest extends TestCase
 {
     private $dataFilePath = __DIR__.'/../../src/DataSources/mledoze/countries/dist/countries.json';
 
-    protected function setUp()
+    protected function setUp(): void
     {
         if (file_exists($this->dataFilePath)) {
             rename($this->dataFilePath, $this->dataFilePath.'_tmp');
         }
     }
 
-    protected function tearDown()
+    protected function tearDown(): void
```

Comment:
To be compatible with `PHPUnit\Framework\TestCase::tearDown`, it should add `:void` type hint.